code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -217,3 +217,17 @@
 		B.deity_name = "Success"
 		B.icon_state = "greentext"
 		B.item_state = "greentext"
+
+/obj/item/weapon/storage/box/syndie_kit/assault_box
+	name = "A box. An unassuming box of WAR!"
+
+/obj/item/weapon/storage/box/syndie_kit/assault_box/New()
+	..()
+	new /obj/item/weapon/storage/backpack/duffel/syndie/ammo/assault(src)
```

Comment:
well an empty dufflebag could easily fit! not to mention realism in what people put in boxes isent really a thing or you wouldent have alot of items